backend/tests/test_sector_etf_map.py

```
@@ -0,0 +1,147 @@
+
+
+def test_sector_etf_map_has_eleven_entries():
+    assert len(SECTOR_ETF_MAP) == 11
+
+
+def test_sector_etf_map_matches_frozen_contract():
+    """Every plan-§3.3 entry maps to the frozen ticker. No extra entries."""
+    assert SECTOR_ETF_MAP == _EXPECTED
+
+
+@pytest.mark.parametrize("sector,etf", list(_EXPECTED.items()))
```

Ruff (0.15.13)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)